CONTRIBUTING.md

```diff
@@ -0,0 +1,138 @@
+Contributing to Sherpa
+======================
+
+Thanks for reporting an issue, fixing a bug, or adding a new feature to
+Sherpa. The following guidelines are not meant to be too onerous; please open
+an issue if you find this to be wrong! Thanks are due to the [AstroPy
+project](http://www.astropy.org/), from which we based this document.
+
+Reporting Issues
+----------------
+
+When [opening an issue](https://github.com/sherpa/sherpa/issues) to report a
+problem, please try and provide a minimal code example that reproduces the
+issue, and also include details of the operating system, and the versions of
+the code you are using - in particular for Sherpa, NumPy, and Python. The
+Sherpa version can be found using
+
+    $ python -c 'import sherpa; print(sherpa.__version__)'
+    40701
```

Comment:
Maybe we should include a more current standalone example rather than a CIAO version?